iphone/Classes/KrollContext.m

```diff
@@ -1177,11 +1179,18 @@ - (void)main
   [kroll release];
   TiStringRelease(prop);
 
-  [self bindCallback:@"setTimeout" callback:&SetTimeoutCallback];
+#ifdef USE_JSCORE_FRAMEWORK
+  JSContext *jsContext = [JSContext contextWithJSGlobalContextRef:context];
+  [JSTimerManager initializeInContext:jsContext];
+#else
+  [self bindCallback:@"setTimeout"
+            callback:&SetTimeoutCallback];
```

Comment:
No idea why it suddenly wants to format it like this. Total non-sense if you ask me. If you remove the `#ifdef` the break is not required anymore.